addon-resizer/Makefile

```diff
@@ -21,12 +21,12 @@ ALL_ARCH = amd64 arm arm64 ppc64le s390x
 ARCH ?= amd64
 
 GOARM=7
-GOLANG_VERSION = 1.12.1
+GOLANG_VERSION = 1.14
```

Comment:
is there a reason this doesn't use a current version like go1.16? that's the version client-go v0.21.0 is built/tested with

Reply:
Updated (go.mod too).